data/render/src/lib.rs

```diff
@@ -21,6 +23,50 @@ pub fn validate(tags: &Tags, entries: &Vec<Entry>) -> Result<(), Box<dyn Error>>
     Ok(())
 }
 
+#[tokio::main]
+pub async fn check_deprecated(token: std::string::String, entries: &mut Vec<Entry>) -> Result<(), Box<dyn Error>> {
```

Comment:
`std::string::String` is part of the Rust prelude, which is a fancy way of saying you can just use `String` here without importing anything. 😎